hirola/front/templates/front/phone_profile.html

```diff
@@ -135,12 +135,13 @@ <h5 id="price" class="green-text">{{ phone.currency }} {{ phone.price|intcomma }
                                 <div class="col s12">
                                         <div class="row">
                                             <div class="input-field col s12">
-                                                <button class="btn waves-effect waves-light green" id="add-to-cart-button" type="button">Buy Now</button>
+                                                <button class="btn waves-effect waves-light green" id="add-to-cart-button" name="buy_now" type="submit" value="1">Buy Now
```

Comment:
Hey, I am not sure what the `value="1"` adds to the functionality of the buy now button. I think it will still function without it.

Reply:
it can't, I've tried it locally and this guys insist where i found this solution https://stackoverflow.com/questions/21505255/identify-which-submit-button-was-clicked-in-django-form-submit, i guess you need to have it so that the name of the buttons can be set which is what you check against so that you can decide where to go next